config/zones/PT-MA.yaml

```diff
@@ -78,6 +78,8 @@ emissionFactors:
       value: 12.62
 region: Europe
 sources:
+  Production:
+    link: https://www.eem.pt/pt/conteudo/sustentabilidade/rotulagem-de-energia-eletrica/evolucao-mensal-do-mix-da-oferta-e-das-emissoes-de-co2/
```

Comment:
This sources section is just for emission factors, the sources for production is gotten directly from flowtracing and the source data tables we have.

normally this would be from the parsers, but I think silke was working on adding it to GPZD as well.